ReactQt/runtime/src/reactuimanager.cpp

```diff
@@ -120,6 +120,20 @@ void ReactUIManager::updateView
   m_bridge->visualParent()->polish();
 }
 
+void ReactUIManager::setChildren
+(
+  int containerReactTag,
+  const QList<int>& childrenTags
+)
+{
+  //This is a simple implementation which fixes a broken example. It's not properly tested and may need revisiting
+  QList<int> newIndices;
+  foreach(int i, childrenTags) {
+     newIndices.append(i + 1);
```

Comment:
@narek-h it seems spacing formatting differs here from rest of the project, 2 spaces indents should be used